src/components/Integration.tsx

```diff
@@ -528,17 +474,29 @@ export function IntegrationPanel(props: IProps) {
         const langTag =
           proj && proj.attributes ? proj.attributes.language : undefined;
         if (offline) {
-          getParatextDataPath().then((ptPath) =>
-            getLocalProjects(ptPath, t.projectsPending, langTag)
+          const localprojs: ProjectIntegration[] = projectintegrations.filter(
+            (pi) =>
+              related(pi, 'integration') === paratextIntegration &&
+              pi.attributes
           );
+          var projIds = localprojs.map((pi) => {
+            var settings = JSON.parse(pi.attributes.settings);
+            return {
+              Name: settings.Name,
+              Id: remoteIdNum('project', related(pi, 'project'), memory.keyMap),
```

Comment:
I think we won't want to use remoteIdNum for the local projects because we need it to work when the program is in offline only mode and when remoteIdNum always returns undefined.

Reply:
yes, good catch.